example/src/Client/utils/AccountsManager.ts

```diff
@@ -52,6 +53,7 @@ export class AccountsManager {
   /** Public key => AccountData */
   private accounts: Map<string, AccountData> = new Map();
   private events: EventEmitter = new EventEmitter();
+  private synced = false;
```

Comment:
question: I don't see where this is getting set to true. Is this being used?

Reply:
https://github.com/iron-fish/wallet-server/pull/27/files/a0948bb73d40f42c1d0e28a804d9f4d029705a39#diff-100d016004f580d38d2fcdcc9afbc0c4fe0693099a7e78ce238a0c66ba6696d0R93